kubernetes/config-maps/nginx-proxy/nginx.conf.j2

```diff
@@ -0,0 +1,40 @@
+user  nginx;
+worker_processes  1;
+
+error_log  /var/log/nginx/error.log warn;
+pid        /var/run/nginx.pid;
+
+
+events {
+    worker_connections  1024;
+}
+
+
+http {
+    include       /etc/nginx/mime.types;
+    default_type  application/octet-stream;
+
+    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+                      '$status $body_bytes_sent "$http_referer" '
+                      '"$http_user_agent" "$http_x_forwarded_for"';
+
+    access_log  /var/log/nginx/access.log  main;
+
+    sendfile        on;
+    keepalive_timeout  65;
+
+    include /etc/nginx/configs/*http.conf;
+}
+
+
+# TCP connection
+stream {
+
+    log_format basic '$remote_addr [$time_local] '
+                     '$protocol $status $bytes_sent $bytes_received '
+                     '$session_time';
+
+    access_log /var/log/nginx/access.log basic buffer=32k;
+
+    include /etc/nginx/configs/*tcp.conf;
```

Comment:
Please include only `tcp.conf` (without the asterisk).

Reply:
Done.